shema_3.sql

```diff
@@ -0,0 +1,20 @@
+create TABLE games(
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    multiplayer BOOLEAN,
+    last_played_at DATE,
+    publish_date DATE,
+    archived BOOLEAN,
+    genre_id INTEGER,
+    FOREIGN KEY(genre_id) REFERENCES genre(id),
+    author_id INTEGER,
+    FOREIGN KEY(author_id) REFERENCES author(id)
+    source_id INTEGER,
+    FOREIGN KEY(source_id) REFERENCES source(id)
+    author_id INTEGER,
+    FOREIGN KEY(author_id) REFERENCES author(id)
+);
+create TABLE author(
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    first_name VARCHAR(255),
+    last_name VARCHAR(255),
+);
```

Comment:
I think it is best to add another table ( A JOINT TABLE) where you can link author to item since an author may have many items inside and the only way to make sure of the relation happens that is one to many you need a join table between author and item so for exmaple : 
author_item table with these properties:
author_id: foreign key
item_id: foreign key
id: unique key

Reply:
Thanks Man

Reply:
welcome